common/src/main/java/com/microsoft/identity/common/adal/internal/tokensharing/TokenShareUtility.java

```diff
@@ -232,7 +234,25 @@ public void saveMsaFamilyRefreshToken(@NonNull final String refreshToken) throws
                 sBackgroundExecutor.submit(new Callable<Pair<MicrosoftAccount, MicrosoftRefreshToken>>() {
                     @Override
                     public Pair<MicrosoftAccount, MicrosoftRefreshToken> call() throws ClientException {
-                        final ADALTokenCacheItem cacheItemToRenew = createTokenCacheItem(refreshToken);
+                        // Use the /consumers endpoint relative to the current cloud
+                        final Uri defaultAuthorityUri = Uri.parse(mDefaultAuthority);
```

Comment:
Just calling out that this logic assumes the default authority takes the form of:
```
https://<host>.com/<tenant_or_tenant_alias>
```